src/Core/Checkout/Cart/LineItemFactoryRegistry.php

```diff
@@ -84,6 +85,7 @@ public function update(Cart $cart, array $data, SalesChannelContext $context): v
             $lineItem->setQuantity($data['quantity']);
 
             $this->eventDispatcher->dispatch(new LineItemQuantityChangedEvent($lineItem, $cart, $context));
```

Comment:
@philipgatzka do you think, we need a deprecation message here, too? that this event dispatching will be removed? or is it sufficient, that the event class is deprecated?

Reply:
Good point, another deprecation comment would be nice to have I think, since it would be visible to developers independent of the IDE (-features) they're using.
```suggestion
            /** @deprecated tag:v6.4.0 - The LineItemQuantityChangedEvent will be removed in the future, please use the BeforeLineItemQuantityChangedEvent and AfterLineItemQuantityChangedEvent variants of this event going forward */
            $this->eventDispatcher->dispatch(new LineItemQuantityChangedEvent($lineItem, $cart, $context));
```